XRTK.SDK/Packages/com.xrtk.sdk/Profiles~/Input/DataProviders/HandControllers/Simulated/Left/YawCounterClockwiseProfile.asset.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: b7a023ed9a50a624691ad0b5a440cb3c
+guid: 17f3fbe848f439d42b507e772d4815f2
```

Comment:
Looks like a lot of guids go regenerated. Any chance we can revert them?

Reply:
In this case it's fine. The underlying script has changed and the purpose of the asset as well.